topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.md

```diff
@@ -225,84 +225,15 @@ The mzidentml output from the Peptide shaker is converted into an sqlite databas
 >
 {: .hands_on}
 
-The next step is to remove known peptides from the list of PSM's that we acquired from the Peptide Shaker results. For that we need to perform some text manipulation steps to extract list of known peptides from the UniProt and cRAP database.
+The next step is to remove known peptides from the list of PSM's that we acquired from the Peptide Shaker results. For that we need to perform Query tabular to extract list of known peptides from the UniProt and cRAP database.
```

Comment:
btw, nice streamlining, always good to be able to get rid of a bunch of text manipulation steps :+1:

Reply:
I think Smorgasbord was a good idea...our tutorials got updated that way 😉